cmd/agent/subcommands/streamlogs/command.go

```diff
@@ -33,18 +37,30 @@ type cliParams struct {
 	filters diagnostic.Filters
 }
 
+type StreamLogParams struct {
+	// Output represents the output file path to write the log stream to.
+	FilePath string
+
+	// Duration represents the duration for which log stream will run
+	Duration time.Duration
```

Comment:
Unless I am missing something, duration doesn't appear to be used anywhere.

Reply:
Oh I saw your comment that this is not used yet. 
I think we should only include functioning code. So I would suggest removing this for now and including it in the next PR when it is used.

Reply:
ill go ahead and remove duration for this pr then 👍🏼